elements/pfe-sass/maps/_typography.scss

```diff
@@ -81,3 +81,148 @@ $pf-type-sizing--titles: (
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    ),
+    "2xl": (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-2xl--FontSize),
+      LineHeight: map-get($pfe-typography-base, line-height--heading),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    ),
+    xl: (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-xl--FontSize),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    ),
+    lg: (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-lg--FontSize),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    ),
+    md: (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-md--FontSize),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    ),
+    sm: (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-sm--FontSize),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    ),
+    xs: (
+      FontFamily: map-get($pfe-typography-base, font-family--heading),
+      FontSize:   map-get($pf-type-sizing--titles, title--m-xs--FontSize),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--normal),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--sm),
+    )
+  ),
+  text: (
+    xl: (
+      FontFamily: map-get($pfe-typography-base, font-family),
+      FontSize:   map-get($pf-type-sizing, FontSize--xl),
+      LineHeight: map-get($pfe-typography-base, line-height),
+      FontWeight: map-get($pfe-typography-base, font-weight--light),
+      content-spacer: map-get($pfe-vars, content-spacer--heading--md),
+    ),
+    // @TODO Update this to match above; this is a temp test for the pfe-text mixin
+    md: (
+      FontSize: pfe-var(content--FontSize)
+    ),
+    sm: (
+      FontSize: pfe-var(content--small--FontSize)
+    )
+  ),
+  /// The variable stack includes a hook just for headings when they appear in the 
```

Comment:
This was such useful information so I wanted to keep that somewhere so I added it here where the hook shows up now